### ISSUE TITLE
vscode debugging

### ISSUE THREAD

**Issue by butcheryl** [ds300/react-native-typescript-transformer#53]
Hi, David

I use  [vscode-react-native](https://github.com/Microsoft/vscode-react-native) debugging

How to add breakpoint in vscode? 

vscode tip me  ` map not found` ？

Thanks

**Comment by ds300** [comment]
Hi! Thanks for the report! :)

I tried it and found a similar error, so it's not just you at least. The normal react-native debugging works (I tried with the chrome devtools or the react-native debugger app on macOS), so i can only assume it's down to a difference with how the vscode debugger hooks itself up to the packager.

Unfortunately I don't have time to look deeper into this for you. Please report back if you find a solution!

**Comment by Keyj1n** [comment]
Hello there,

i have sadly the same problem.
If you find any solution, please let me know.

Thanks and Kind Regards,
Keyjin

**Comment by jwargo** [comment]
Just like with any other IDE, you simply click in the margin to set the breakpoint.  You can also highlight the line and press the F9 key.

**Comment by jsslai** [comment]
Debugging is working for me but sometimes I'm unable to set breakpoint to a specific line and it might jump elsewhere. Do you know what is causing that?

**Comment by animaonline** [comment]
@jsslai I'm having the same issue, I think it has something to do with the source maps not being generated correctly. Were you able to solve your problem?

**Comment by jsslai** [comment]
@animaonline  No. I usually add extra console.log rows until VS code is able to add the breakpoint. :D
